### PR TITLE
Disable Claim rewards when non pending rewards

### PR DESCRIPTION
**Pull request by victorcoder**
The button allow to execute a contract call that won't do anything.

### CODE REVIEW

**Review by Macxim — CHANGES_REQUESTED** [review]
Good catch!
Could you maybe just hide the button if there are no rewards?

Screen Shot 2021-08-23 at 9 08 51 PM

**Comment by victorcoder** [comment]
Feedback applied

File: web/components/mint.tsx [hunk 1]
```diff
@@ -388,15 +387,16 @@ export const Mint = () => {
...
-              <button 
-                type="button" 
-                className={classNames(pendingVaultRewards === 0 && 'hidden',
-                  'bg-indigo-600 hover:bg-indigo-700 inline-flex items-center px-3 py-2 ml-4 text-sm font-medium leading-4 text-white border border-transparent rounded-md shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500')}
-                onClick={() => claimPendingRewards()}
-                disabled={pendingVaultRewards === 0}
-              >
-                Claim rewards
-              </button>
+              { pendingVaultRewards > 0 &&
```
Comment (Macxim) [inline web/components/mint.tsx:390]: Extra space before `{` here.
Also, we used the ternary `{condition ? () : null}` throughout the code base, maybe we should stick to it? I'm no expert, just aiming for consistency here.

Comment (philipdesmedt) [reply web/components/mint.tsx:390]: I guess we don't need an `else` clause, probably remove the `null`s in other places

Comment (victorcoder) [reply web/components/mint.tsx:390]: Agree with @philipdesmedt , this construct is better that ternaries with `null`, and to achieve consistency it's better to refactor the other places.

Comment (Macxim) [reply web/components/mint.tsx:390]: Can you explain how/why it's better? https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Comment (victorcoder) [reply web/components/mint.tsx:390]: JS wonderfulness 😂 , that's a compelling reason to not use the `&&` then. I'll apply the changes.

**Comment by victorcoder** [comment]
Feedback applied, let me know if you want me to squash commits or you are going to to Squash and merge the PR.

**Comment by philipdesmedt** [comment]
@victorcoder thanks mate! We just squash and merge here so no need :)

**Comment by Macxim** [comment]
@victorcoder Good job! Thanks!

**Pull request merged**